paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetActivityTest.kt

```diff
@@ -235,8 +227,6 @@ internal class CustomerSheetActivityTest {
 
     @Test
     fun `When customer ACHv2 is disabled, should display form elements`() {
```

Comment:
I would think this should fail after this change.

Why is it not?

Reply:
It won't because we removed the flag check in #8142 so the form elements are always displayed irregardless of the feature flag state. These tests are irrelevant now that we have removed the flag though so I can remove them.

Reply:
At the very least rename the test if it's relevant.